mutator/expression/comparison.go

```diff
@@ -0,0 +1,47 @@
+package expression
+
+import (
+	"go/ast"
+	"go/token"
+	"go/types"
+
+	"github.com/zimmski/go-mutesting/mutator"
+)
+
+var tokenMutations = map[token.Token]token.Token{
+	token.LSS: token.LEQ,
+	token.LEQ: token.LSS,
+	token.GTR: token.GEQ,
+	token.GEQ: token.GTR,
+}
+
+func init() {
```

Comment:
Please move the "init" before "tokenMutations" to make it clearer which mutators are registered in this file.